packages/functional-tests/tests/settings/password.spec.ts

```diff
@@ -57,3 +57,87 @@ test.describe('severity-1 #smoke', () => {
     expect(status).toEqual('Not Set');
   });
 });
+
+test.describe('password strength tests', () => {
+  test.beforeEach(
+    async ({ target, credentials, page, pages: { login } }, { project }) => {
+      test.slow(project.name !== 'local', 'email delivery can be slow');
+      const email = login.createEmail('sync{id}');
```

Comment:
I think you can created a regular email and not use sync. Also, you should be able to load the default page via

```
await page.goto(target.contentServerUrl, { waitUntil: 'networkidle' });
```